packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -71,9 +194,34 @@ const BoldChunk = (chunks) => <Typography fontWeight="bold">{chunks}</Typography
  * SelectedEntriesModalContent
  * -----------------------------------------------------------------------------------------------*/
 
-const SelectedEntriesModalContent = ({ onToggle, onConfirm }) => {
+const SelectedEntriesModalContent = ({ onToggle, onConfirm, onRefreshData }) => {
   const { formatMessage } = useIntl();
-  const { selectedEntries } = useTableContext();
+  const { selectedEntries, rows, isLoading } = useTableContext();
+  const { contentType, components } = useSelector(listViewDomain());
+
+  /**
+   * @returns {{validIds: number[], errors: Object.<number, string>}} - Returns an object with the valid ids and the errors
+   */
+  const validateEntriesToPublish = React.useCallback(() => {
+    const validations = { validIds: [], errors: {} };
+    // Create the validation schema based on the contentType
+    const schema = createYupSchema(contentType, { components }, { isDraft: false });
+    // Validate each entry
+    rows.forEach((entry) => {
+      try {
+        schema.validateSync(entry, { abortEarly: false });
+        validations.validIds.push(entry.id);
+      } catch (e) {
+        validations.errors[entry.id] = getYupInnerErrors(e);
+      }
+    });
+
+    return validations;
+  }, [rows, contentType, components]);
+
+  const { errors } = validateEntriesToPublish();
```

Comment:
> but I need to create a state for the errors on the BulkActionButtons component and pass that through two levels to the SelectedEntriesModalContent to use it, and now I need also the validIds, so with that it would be 2 states that I need to pass as props, and also we will need to manually run the function when user refresh the data

Yeah I think some of that is the way we've structured things and maybe we can find a better solution (another PR). See my other commen on SelectedEntriesModal

Reply:
I think the third option would be to run this code here in this component in a useEffect based on the isSelectedEntriesModalOpen state from BulkActionButtons. I don't really see anything wrong with what you have, but tbh I'm not really sure what the _best_ option is. 

Maybe @joshuaellis has some input?